doc/user_guide.md

```diff
@@ -249,10 +249,9 @@ Refer to spdm_client_init() in [spdm_requester.c](https://github.com/DMTF/spdm-e
 
 6. Send and receive message in an SPDM session
 
-   6.1, Use the SPDM vendor defined message.
-        (SPDM vendor defined message + transport layer header (SPDM) => application message)
+   6.1, Use the SPDM vendor defined request.
    ```
-   libspdm_send_receive_data (spdm_context, &session_id, FALSE, &request, request_size, &response, &response_size);
+   libspdm_vendor_request (spdm_context, standard_id, vendor_id_len, &vendor_id, &request, request_size, &response, &response_size);

```

Comment:
Do you want to send request only, or you want to send request and get response?

If later, then the function name should reflect that, e.g. libspdm_vdm_send_receive_data().

Reply:
I want to emphasize it is specifically for vendor defined requests, as opposed to generic messages. Would libspdm_vendor_request_and_response() work better?